src/main/kotlin/com/openlattice/shuttle/MissionControl.kt

```diff
@@ -274,7 +274,11 @@ class MissionControl(
             propertyTypes = edmApi.propertyTypes.map { it.type to it }.toMap().toMutableMap()
             propertyTypesById = propertyTypes.mapKeys { it.value.id }
         } catch ( thrown: Throwable ) {
-            MissionControl.fail(1, Flight.newFlight().done(), thrown)
+            MissionControl.fail(
+                    1,
+                    throw IllegalStateException("Could not load EDM elements.  Check your authentication."),
```

Comment:
I think this error message will be misleading most of the time since this is inside a `} catch (throw: Throwable ){`, which will print this error message for any and all kinds of errors. 
I think it would be better in a separate catch block but, while I'm sure it exists, I'm not sure what subset of `Throwable` you want to print this for

Reply:
Also there's no need to create a new IllegalStateException here, you can just call `logger.error("Could not load EDM elements.  Check your authentication.")` before you call `MissionControl.fail` and pass `thrown` into `MissionControl.fail`. That way, the stacktrace printed in `MissionControl.fail` will point to the exact error that occurred while if you throw `IllegalStateException` here, the stacktrack will point to this line (279) instead of straight to the actual error